tests/net/skald_bthome/main.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2025 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       SAUL-less skald_bthome tests
+ *
+ * @author      Martine S. Lenders <martine.lenders@tu-dresden.de>
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "net/netdev/ble.h"
+#include "net/skald/bthome.h"
+#include "od.h"
+#include "ztimer.h"
+
+#define BTHOME_ADV_INTERVAL           (10U)
+
+static skald_bthome_ctx_t _ctx;
+static netdev_t _mock_nrfble_dev;
+static const netdev_driver_t _mock_netdev_driver;
+
+netdev_t *nrfble_setup(void)
+{
+    _mock_nrfble_dev.driver = &_mock_netdev_driver;
+    _mock_nrfble_dev.event_callback = NULL;
+    _mock_nrfble_dev.context = NULL;
+    return &_mock_nrfble_dev;
+}
+
+static int _mock_nrfble_init(netdev_t *dev)
+{
+    puts("[mock_nrfble] initialization successful");
+
+    /* signal link UP */
+    dev->event_callback(dev, NETDEV_EVENT_LINK_UP);
+
+    return 0;
+}
+
+static int _mock_nrfble_send(netdev_t *dev, const iolist_t *data)
+{
+    (void)dev;
+    assert(data);
+
+    for (unsigned i = 0; data; i++, data = data->iol_next) {
+        printf("Data item %u sent:\n", i);
+        od_hex_dump(data->iol_base, data->iol_len, OD_WIDTH_DEFAULT);
+    }
+
+    return 0;
+}
+
+static int _mock_nrfble_get(netdev_t *dev, netopt_t opt, void *val, size_t max_len)
+{
+    (void)dev;
+    (void)max_len;
+
+    switch (opt) {
+        case NETOPT_DEVICE_TYPE:
+            assert(max_len >= sizeof(uint16_t));
+            *((uint16_t *)val) = NETDEV_TYPE_BLE;
+            return sizeof(uint16_t);
+        default:
+            return -ENOTSUP;
```

Comment:
```suggestion
    case NETOPT_DEVICE_TYPE:
        assert(max_len >= sizeof(uint16_t));
        *((uint16_t *)val) = NETDEV_TYPE_BLE;
        return sizeof(uint16_t);
    default:
        return -ENOTSUP;
```

I think there is a coding convention to keep the case at the same indentation as the switch. This is not explicitly mentioned in the document, but can be seen here: https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#indentation-of-preprocessor-directives